### DISCUSSION TITLE
Version 4.00 release

### DISCUSSION THREAD

**Discussion by fritzrehde** [transmission/transmission#3953] [General]
Just out of curiousity, when do you think version 4.00 will come out of beta and will be released properly?

**Comment by nevack** [comment]
> Just out of curiousity, when do you think version 4.00 will come out of beta and will be released properly?

As soon as all bugs are squashed 😄 
You can help us by testing beta version 🥇

**Reply by juliand665** [reply]
How do I install the beta? I have the checkbox checked but am not seeing any updates since 3.00 over two years ago 🤔 
<img width="309" alt="image" src="https://user-images.githubusercontent.com/827762/195965659-f0d88127-bb09-461b-bb5b-4d558a25e2ab.png">

**Comment by GaryElshaw** [comment]
Nightly builds can be found here for each platform (grey Links): https://transmissionbt.com/download

**Reply by juliand665** [reply]
Thank you! So glad to finally have the new icon and UI :)

**Comment by GaryElshaw** [comment]
@ckerr Charles, can sparkle be activated now that Transmission is in beta?

**Reply by spupuz** [reply]
can be 4.0 tested in docker is there an image already?
